extensions/adobe/experience/privacy.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+	"type": "object",
+	"title": "Privacy",
+	"meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/consent"],
+	"description": "Privacy Mixin.",
+	"definitions": {
+		"privacy": {
+			"type": "object",
+			"properties": {
+				"xdm:consentAssertionId": {
+					"title": "Assertion Id",
+					"description": "Consent assertion ID.",
+					"type": "string"
+				},
+				"xdm:consentStack": {
+					"title": "Consent Stack",
+					"type": "string",
+					"description": "Consent Stack.",
+					"meta:enum": {
+						"General_Opt_Out": "General_Opt_Out",
+						"Sell_Share": "Sell_Share"
+					}
+				},
+				"xdm:consentPurpose": {
+					"title": "Consent Purpose",
+					"type": "string",
+					"description": "User consent for a Consent Purpose within a given consent Stack.",
+					"meta:enum": {
+						"Opt_Out": "Opt_Out"
+					}
+				},
+				"xdm:consentSource": {
+					"title": "Consent Source",
+					"type": "string",
+					"description": "Source Application of User consent- Mobile Web,Mobile App,Web.",
+					"meta:enum": {
+						"mobile_Web": "mobile_Web",
+						"mobile_App": "mobile_App",
+						"web": "web"
+					}
+				},
+				"xdm:version": {
```

Comment:
What is the implication of version field? Is UPS supposed to keep multiple versions of same consent structure?

Reply:
The "version" filed is a requirement as per the PRD. If its provided by the CMP system, XDM should preserve that. The question of the usage could be answered by the business or the PM team ﻿Karthik Raman﻿